Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -6417,3 +6424,42 @@ void MainWindow::slotEnableVoiceActivation(bool checked)
 {
     slotMeEnableVoiceActivation(checked, SOUNDEVENT_VOICEACTMEON, SOUNDEVENT_VOICEACTMEOFF);
 }
+
+#ifndef Q_OS_LINUX
+bool MainWindow::eventFilter(QObject *object, QEvent *event)

```

Comment:
Isn't it simpler to override QWidget::keyPressEvent() like we do in TTSEventsTreeView-class. It seems excessive to override eventFilter() since this handles _all_ events. Also non-key press.

Reply:
Hmmmm, with overriding keyPressEvent, we can't apparently not check what object receive the key, or I don't know do this... So it's maybe not applicable here.